src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/model/EmailContent.java

```diff
@@ -15,6 +18,8 @@
         private String emailAddress;
         private String createdAt;
 
+        private final ObjectMapper objectMapper = new ObjectMapper();
+
         public Builder() {
```

Comment:
You shouldn't need this empty argument constructor - Java provides it by default.

Reply:
Didn't add this, passed previous PR but removed empty constructor.

Reply:
Yep, appreciate that. Thanks 👍 .